blackjax/__init__.py

```diff
@@ -5,6 +5,7 @@
     elliptical_slice,
     hmc,
     mala,
+    marginal_latent_gaussian,
```

Comment:
Should we just go with `mGrad` like in the paper?

Reply:
It's not very standard...

Reply:
That being said, if you want to change the name, please do. I'm not attached to it.